File: ApiSuvesaPos/Negocio/Modelo/inventario/ajustesinventario/ajusteinventario.vb
Namespace Modelo
    Public Class ajusteinventario
        Public Property Consecutivo As Long
        Public Property Fecha As DateTime
        Public Property Anula As Boolean
        Public Property Cedula As String
        Public Property TotalEntrada As Decimal
        Public Property TotalSalida As Decimal
        Public Property SaldoAjuste As Decimal
        Public Property Asiento As Decimal
        Public Property Contabilizado As Boolean
        Public Property AjusteInventario_Detalle As New List(Of ajusteinventario_detalle)
    End Class

End Namespace

File: ApiSuvesaPos/ApiSuvesaPos/Modelo/inventario/ajustesinventario/ajusteinventario.cs
namespace Modelo
{
    public class ajusteinventario
    {
        public long Consecutivo { get; set; }
        //public DateTime Fecha { get; set; }
        public bool Anula { get; set; }
        public string Cedula { get; set; }
        public decimal TotalEntrada { get; set; }
        public decimal TotalSalida { get; set; }
        public decimal SaldoAjuste { get; set; }
        public decimal Asiento { get; set; }
        public bool Contabilizado { get; set; }
        //public List<ajusteinventario_detalle> AjusteInventario_Detalle { get; set; } = new List<ajusteinventario_detalle>();
    }
}
